### ISSUE TITLE
RBAC: Google: Support google groups

### ISSUE THREAD

**Issue by Haarolean** [kafbat/kafka-ui#55]
via https://github.com/provectus/kafka-ui/issues/3279

**Comment by jaskeerat789** [comment]
Hey kafbat community, is this there in kafbat's roadmap?

**Comment by Haarolean** [comment]
> Hey kafbat community, is this there in kafbat's roadmap?

nope, google doesn't expose required APIs properly, making this really complicated

**Comment by jaskeerat789** [comment]
> nope, google doesn't expose required APIs properly, making this really complicated

I can understand this. Could SAML be an option here since it does come with membership attributes? I see that as of now only OAuth2 and LDAP is supported.

**Comment by germanosin** [comment]
@jaskeerat789 Unfortunately, SAML support has not yet been implemented in Spring Security. See: https://github.com/spring-projects/spring-security/issues/7954

**Comment by younsl** [comment]
Is there any progress on this feature? I have a requirement to control RBAC based on Google Groups API rather than individual user emails when using Google OAuth2.
